ui.apps/src/main/content/jcr_root/conf/venia/settings/wcm/policies/.content.xml

```diff
@@ -27,18 +27,14 @@
                         mergeList="{Boolean}true"/>
                     <default
                         jcr:description="Policy of the default root parsys"
+                        jcr:lastModified="{Date}2019-05-09T11:14:56.461+05:30"
+                        jcr:lastModifiedBy="admin"
                         jcr:primaryType="nt:unstructured"
                         jcr:title="Default Layout Container"
                         sling:resourceType="wcm/core/components/policy/policy"
-                        components="[/libs/wcm/foundation/components/image,/libs/wcm/foundation/components/responsivegrid,group:AEM.Venia]">
+                        components="[/libs/wcm/foundation/components/image,/libs/wcm/foundation/components/responsivegrid,group:AEM.Venia,group:Commerce]">
                         <cq:authoring jcr:primaryType="nt:unstructured">
                             <assetToComponentMapping jcr:primaryType="nt:unstructured">
-                                <product
```

Comment:
I think we have to clarify if the drag'n'drop should create a `Product` or `ProductTeaser` component, I'm not sure this has clearly been defined by product management.

Reply:
I think it should be `ProductTeaser` as this is the component to be used by the authors while the other one is for the PDP.

Reply:
I think dropping a `product` asset should not automatically create anything. Authors would expect that DnD would create a product component, as with the other assets - images create `image` components, videos create `video` asset components, experience fragments create `experiencefragment` components...

Reply:
As discussed , leaving it to be <code>ProductTeaser</code>.